tracer/test/Datadog.Trace.TestHelpers/MockSpanEvent.cs

```diff
@@ -0,0 +1,23 @@
+// <copyright file="MockSpanEvent.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using MessagePack;
+
+namespace Datadog.Trace.TestHelpers
```

Comment:
[The question I asked there](https://github.com/DataDog/dd-trace-dotnet/pull/6769/files#r2018319214) would apply here as well 😁